packages/router/src/router_config.ts

```diff
@@ -85,8 +85,11 @@ export interface RouterConfigOptions {
 
   /**
    * Defines how the router merges parameters, data, and resolved data from parent to child
-   * routes. By default ('emptyOnly'), inherits parent parameters only for
-   * path-less or component-less routes.
+   * routes.
+   *
+   * By default ('emptyOnly'), a route A inherits parent route's B parameters, when A has an empty
+   * path (meaning its configured with path: '') or when B doesn't have any component
+   * set.
```

Comment:
```suggestion
   * By default ('emptyOnly'), a route inherits the parent route's parameters when the route itself has an empty
   * path (meaning its configured with path: '') or when the parent route doesn't have any component
   * set.
```

I think it would be best not to alias the current and parent routes as A and B, respectively.

Reply:
I fear that without it, people still may misunderstand it, but lets go with your suggestion.

Reply:
https://github.com/angular/angular/pull/52170/commits/085684611077d8bca76b51096c152be8898c2684